index.html

```diff
@@ -68,6 +68,8 @@
          * it */
         if (target) {
             var qr = new QRCode(target, {
+                width : 500,
```

Comment:
Is there a particular reason to hardcode this in pixels? Does qrcode.js not support `vh` and `vw` units?